db/re/item_db.txt

```diff
@@ -13121,10 +13539,17 @@
 31610,C_Yellow_Wizardry_Hat,C Yellow Wizardry Hat,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,286,{},{},{}
 31611,C_Snake_Lord_Stole,C Snake Lord Stole,4,0,,,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,1810,{},{},{}
 31612,C_Large_Sorcerer_Crown,C Large Sorcerer Crown,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1811,{},{},{}
+31614,C_Fox,Costume Fox,4,,,0,,0,,0,0xFFFFFFFF,63,2,2048,,,,1812,{},{},{}
+31616,C_Counting_Sheep,Costume Counting Sheep,4,,,0,,0,,0,0xFFFFFFFF,63,2,4096,,,,1814,{},{},{}
+31624,C_Cat_Heart,Costume Cat Heart,4,,,0,,0,,0,0xFFFFFFFF,63,2,1024,,,,1822,{},{},{}
+31625,C_Protect_Cloth,Coostume Protect Cloth,4,,,0,,0,,0,0xFFFFFFFF,63,2,4096,,,,1761,{},{},{}
+31693,C_Lunatic_Muffler,Costume Lunatic Muffler,4,,,0,,0,,0,0xFFFFFFFF,63,2,4196,,,,1732,{},{},{}
+31699,C_Smiling_Eyes,Costume Smiling Eyes,4,,,0,,0,,0,0xFFFFFFFF,63,2,2048,,,,1880,{},{},{}
 //
 32005,Illusion_Pole_Axe,Illusion Pole Axe,5,0,,1500,190,,3,2,0x00004082,63,2,2,4,120,1,4,{ .@r = getrefine(); bonus bStr,3; bonus bInt,2; bonus bDex,2; if (.@r >= 13) { .@val = -2000; } else if (.@r >= 9) { .@val = -1000; } bonus2 bSkillCooldown,"RK_HUNDREDSPEAR",.@val; },{},{}
 32013,Metal_Stick,Metal Stick,5,0,0,2000,180,,3,3,0x00004080,56,2,2,4,150,1,4,{ .@r = getrefine(); bonus2 bSkillAtk,"LG_CANNONSPEAR",10; if(.@r > 6){ bonus2 bSkillAtk,"LG_OVERBRAND",10 + (.@r > 8 ? 15 : 0); } if(.@r > 11){ bonus bDelayrate,-15; } },{},{}
 32014,Brute_Spear,Brute Spear,5,0,0,,150,,3,0,0x00004080,56,2,2,4,100,1,4,{ .@r = getrefine(); bonus bUnbreakableWeapon; bonus2 bIgnoreDefRaceRate,RC_All,5*.@r; bonus2 bAddRace,RC_Player,80 + (.@r > 8 ? 20 : 0) + (.@r > 7 ? 35 : 0); if(.@r > 9){ bonus bMaxHPrate,15; bonus bMaxSPrate,15; } },{},{}
+32018,Aquatic_Spear_,Aquatic Spear,5,,,1200,180,,3,2,0x00004000,56,2,34,4,100,1,3,{ .@r = getrefine(); if (.@r>=2) { bonus bBaseAtk,.@r/2*10; } if (.@r>=3) { bonus bLongAtkRate,.@r/3*4; } if (.@r>=9) { bonus bAspdRate,10; } if (.@r>=11) { bonus2 bSkillAtk,"LG_CANNONSPEAR",20; bonus2 bSkillAtk,"LG_BANISHINGPOINT",20; } },{},{}
```

Comment:
This item is a One-Handed Spear

So it should change location from 34 -> 2
https://www.divine-pride.net/database/item/32018